cypress/TestCases/FireboltCertification/UserInterest.feature

```diff
@@ -0,0 +1,113 @@
+Feature: UserInterest
+
+    @initialization
+    Scenario: Launch FCA for 'Userinterest'
+        Given the environment has been set up for 'Userinterest' tests
+        And 3rd party 'certification' app is launched
+
+    @coreSDK @sdk @transport @userinterest
+    Scenario Outline: Discovery.userInterest - Positive Scenario: with interest type <Scenario>
```

Comment:
It seems the scenario name and param names looks something off

i.e. 

**_Discovery.userInterest - Positive Scenario: with interest type interest reason playlist_**
**_And '3rd party app' invokes the 'Firebolt' API to 'get userInterest with type interest reason playlist program entity'_**

Not sure how stakeholder would understand this

Reply:
corrected the scenario names